data/json/items/comestibles/veggy_dishes.json

```diff
@@ -656,6 +677,23 @@
     "delete": { "flags": [ "RAW" ] },
     "fun": 2
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "rehydrated_corn",
+    "name": "rehydrated corn kernels",
+    "copy-from": "veggy",
```

Comment:
Not `corn_kernels`?

Reply:
probably a good idea, but I was too lazy to change it back yesterday, I'll do it now